cmd/registry/cmd/compute-details.go

```diff
@@ -176,7 +176,7 @@ func (task *computeDetailsTask) Run() error {
 				Description: description,
 			},
 			UpdateMask: &field_mask.FieldMask{
-				Paths: []string{"owner", "display_name", "description"},
+				Paths: []string{"display_name", "description"},
```

Comment:
The "owner" field was removed from the v1 API - its definition was unclear and it can be represented by a label or annotation. Removing it here and in a few other places fixed a potential crash due to conflicts with the database schema.